src/stylesheets/components/_list-styles.scss

```diff
@@ -108,6 +108,45 @@ dl, dt, dd {
   }
 }
 
+/*--- List Events ---*/
+
+.list--events {
+  @include list;
+  margin-bottom: $base-spacing-unit-medium;
+
+  > li {
+    padding: 0;
+    margin-bottom: 0;
+
+    @include media($desktop) {
+      @include flex(33.333%, 66.666%);
+      align-items: flex-start;
+      border-top: 1px solid $grey-3;
+      border-bottom: 1px solid $grey-3;
```

Comment:
Would be better to only apply a bottom border to the last `li`. That way you can remove the negative margin.